src/main/java/org/coffeetrain/doggifs/MainActivity.kt

```diff
@@ -42,11 +42,18 @@ class MainActivity : Activity() {
         incomingContexts: MutableMap<Any, Context>,
         callback: TraversalCallback) {
       val toScreen = incoming.getKey<Any>();
-      val annotation = checkNotNull(toScreen.javaClass.getAnnotation(Layout::class.java),
-          { "Screen didn't have a layout annotation" })
+      val annotation = checkNotNull(toScreen.javaClass.getAnnotation(Screen::class.java),
+          { "Screen didn't have a layout annotation: " + toScreen.javaClass })
```

Comment:
annotation classes can't have bodies, so it would have to be an extension.  Feels weird, gonna leave as-is.